config/environments/production.rb

```diff
@@ -3,6 +3,8 @@
 require Rails.root.join("config/smtp")
 
 Rails.application.configure do
+  config.force_ssl = true
```

Comment:
should we remove the ssl-related code from `app/controllers/application_controller.rb` ?